### PR TITLE
Rebalance would be triggered if AI distribution is disabled at HC connect

### PR DESCRIPTION
**Pull request by specop0**
**When merged this pull request will:**
- Not trigger a rebalance if AI distribution is disabled and a HC connects

### CODE REVIEW

**Comment by specop0** [comment]
closes #90

File: addons/headless/functions/fnc_handleConnectHC.sqf [hunk 1]
```diff
@@ -28,5 +28,8 @@ if (GVAR(log)) then {
     INFO_1("Registered HC: %1",_headlessClient);
 };
 
+//Exit if AI distribution is disabled
```
Comment (jonpas) [inline addons/headless/functions/fnc_handleConnectHC.sqf]: Space after `//`.

**Comment by jonpas** [comment]
@specop0 If you'd like to test, here is a build of the PBO:
[acex_headless_fix.zip](https://github.com/acemod/ACEX/files/1544883/acex_headless_fix.zip)

**Comment by specop0** [comment]
@jonpas Thanks for the PBO. Rebalancing does not work if disabled, but works if enabled.

File: addons/headless/functions/fnc_handleConnectHC.sqf (hunk 1 above)
Comment (specop0) [reply addons/headless/functions/fnc_handleConnectHC.sqf]: In my master it is fixed...but why isn't the commit added to the PR? o.O
Am I too dumb for github?
https://github.com/specop0/ACEX/commits/master

Comment (jonpas) [reply addons/headless/functions/fnc_handleConnectHC.sqf]: Interesting. 🤔

**Pull request merged**